Terminal.Gui/View/Adornment/Border.cs

```diff
@@ -273,7 +273,7 @@ protected internal override bool OnMouseEvent (MouseEvent mouseEvent)
 
             // Only start grabbing if the user clicks in the Thickness area
             // Adornment.Contains takes Parent SuperView=relative coords.
-            if (Contains (mouseEvent.X + Parent.Frame.X + Frame.X, mouseEvent.Y + Parent.Frame.Y + Frame.Y))
+            if (Contains (new (mouseEvent.X + Parent.Frame.X + Frame.X, mouseEvent.Y + Parent.Frame.Y + Frame.Y)))
```

Comment:
Hm. I'm pretty sure offsetting one of the points by the other two would be less implicit copying.